### PR TITLE
team channel data loading changes

### PR DESCRIPTION
**Pull request by buoyad**
- Put top level team channel stuff into store so components on the same page don't risk getting out of sync with separate states
- Add refresh calls when we make changes

cc @keybase/y2ksquad

### CODE REVIEW

**Comment by buoyad** [comment]
@keybase/react-hackers bump

File: shared/teams/common/channel-hooks.tsx [hunk 1]
```diff
@@ -13,6 +13,24 @@ function filterNull<A>(arr: Array<A | null>): Array<A> {
...
 
+// Filter bots out using team role info, isolate to only when related state changes
+export const useChannelParticipants = (
+  teamID: Types.TeamID,
+  conversationIDKey: ChatTypes.ConversationIDKey
+) => {
+  const participants = Container.useSelector(s => ChatConstants.getParticipantInfo(s, conversationIDKey).all)
+  const teamMembers = Container.useSelector(s => Constants.getTeamDetails(s, teamID).members)
+  const [noBots, setNoBots] = React.useState<Array<string> | undefined>(undefined)
+  React.useEffect(() => {
```
Comment (jakob223) [inline shared/teams/common/channel-hooks.tsx:24]: Why is this a `useEffect`? I feel like `useMemo` would be a lot clearer.

**Comment by buoyad** [comment]
@jakob223 can you try again after rebuilding go? I'm not aware of a specific service change causing this but have hit that and fixed with a rebuilt service.

File: shared/teams/common/channel-hooks.tsx (hunk 1 above)
Comment (buoyad) [reply shared/teams/common/channel-hooks.tsx:24]: Ah thanks, this is wrong, will change.

Comment (buoyad) [reply shared/teams/common/channel-hooks.tsx]: Fixed

**Pull request merged**